File: app/src/main/java/com/gmail/fattazzo/formula1world/domain/F1Season.java
package com.gmail.fattazzo.formula1world.domain;

/**
 * @author fattazzo
 *         <p/>
 *         date: 09/07/17
 */
public class F1Season {

    public Integer year;

    public String url;

    public boolean current;

    public String description;

    @Override
    public String toString() {
        return "F1Season{" +
                "year=" + year +
                ", url='" + url + '\'' +
                ", current=" + current +
                ", description='" + description + '\'' +
                '}';
    }
}

File: app/src/main/kotlin/com/gmail/fattazzo/formula1world/domain/F1Season.kt
package com.gmail.fattazzo.formula1world.domain

/**
 * @author fattazzo
 *
 *
 * date: 09/07/17
 */
class F1Season {

    var year: Int? = null

    var url: String? = null

    var current: Boolean = false

    var description: String? = null

    override fun toString(): String {
        return "F1Season{" +
                "year=" + year +
                ", url='" + url + '\'' +
                ", current=" + current +
                ", description='" + description + '\'' +
                '}'
    }
}
